task/01-strings-tasks.js

```diff
@@ -55,7 +55,7 @@ function getStringLength(value) {
  *   'Chuck','Norris'  => 'Hello, Chuck Norris!'
  */
 function getStringFromTemplate(firstName, lastName) {
-    throw new Error('Not implemented');
+    return new String(`Hello, ${firstName} ${lastName}!`);
```

Comment:
Should return `string` primitive type
String(...) will simply convert the expression into a boolean primitive value, while new String(...) will create a wrapper object around the converted string value